File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        HashMap<Integer,Integer> map = new HashMap<>();
        for(int num : nums)
            map.put(num, map.getOrDefault(num, 0) + 1);

        int maxi = 0;
        for(int value : map.values())
            maxi = Math.max(maxi, value);
        
        int ans = 0;
        for(int value : map.values()) {
            if(value == maxi)
                ans += value;
        }

        return ans;
    }
}

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        int freq[101] = {0}, maxF = 0;
        for(int num : nums) {
            freq[num]++;
            maxF = max(maxF, freq[num]);
        }

        int ans = 0;
        for(int f : freq)
            if(f == maxF)
                ans += f;

        return ans;
    }
};